packages/@pollyjs/adapter-xhr/tests/utils/xhr-request.js

```diff
@@ -22,7 +22,7 @@ export default function request(url, obj = {}) {
       xhr.status === 204 && xhr.responseText === '' ? null : xhr.responseText;
 
     return new Response(responseBody, {
-      status: xhr.status,
+      status: xhr.status || 500,
```

Comment:
I realize this is just pertaining to a test util but can the fallback mask a future bug in our test suite?

Reply:
This will only happen when status is `0` which occurs when we do `xhr.error()`. Do you have a better way around this? Response wont accept a status code of `0`